exporter/f5cloudexporter/README.md

```diff
@@ -31,7 +31,7 @@ Example:
 f5cloud:
   endpoint: https://<ENDPOINT_FOUND_IN_F5_CLOUD_PORTAL>
   source: prod
-  auth:
+  f5cloud_auth:
```

Comment:
Just out of curiosity: are you converting this into an authenticator in the future?

Reply:
Yeah, that the intention. I will be working on an F5 Authenticator as soon as this is merged and will likely be deprecating this exporter in favor of the OTLP gRPC + F5 Auth Ext :)